plans/pea.tf

```diff
@@ -0,0 +1,59 @@
+resource "azurerm_resource_group" "pea" {
```

Comment:
I'm not clear why this is showing up as added. This plan already exists o_O

Perhaps rebase off the master branch again?

Reply:
Indeed I did a bad rebase but it's fixed now